CMakeLists.txt

```diff
@@ -522,8 +522,8 @@ else()
   ENDIF()
 
   find_package(Shapefile) # package libshp-dev
-  if(SHAPEFILE_FOUND)
-    add_dependency_includes(${LIBSHAPEFILE_LIBRARY})
+  if(NOT SHAPEFILE_FOUND)
+    message(FATAL_ERROR "Shapefile library (libshp-dev) was not found")
```

Comment:
yes, but i don't think it must be a requirement. better to have a warning and conditional compilation without time-based conditionals